client/src/components/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./Footer.css";
+import OrganisationLink from "./OrganisationLink";
+import linkedinIcon from "../assets/linkedin-icon.svg";
+import githubIcon from "../assets/github-icon.svg";
+import ExternalLink from "./ExternalLink";
+import instagramIcon from "../assets/instagram-icon.svg";
+
+const Footer = () => {
+    return (
+        <footer>
+            <div className="footer-disclaimer">
+                <p>
+                    Working under the artistic name Unity, Deirdre Molloy will exhibit the Black Atlantic Rhythm Codes map in Nantes, France in 2024. All contributors will be credited, with permission, including the creators of this Code Your Future prototype. The exhibition will be co-produced by Gerador and promoted by Project Manifest.
```

Comment:
I recommend moving all content into a `const content = {}` object to separate copy from code.

E.g.

```js
const content = {
  text: "example"
};

const Component = () => <div>{content.text}</div>;
```

This makes it easier to tell what's code and what's not.